api/src/database/migrations/20211103A-update-special-geometry.ts

```diff
@@ -0,0 +1,27 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex('directus_fields')
+		.update({ special: knex.raw(`REPLACE(special, ',', '.')`) })
+		.whereIn('special', [
+			'geometry,Point',
+			'geometry,LineString',
+			'geometry,Polygon',
+			'geometry,MultiPoint',
+			'geometry,MultiLineString',
+			'geometry,MultiPolygon',
+		]);
```

Comment:
This feels tricky to me. The order of the CSV isn't guaranteed, and people might've added custom things in there as well. I think we should read the values into JS memory first, and then handle it in a map so we can properly filter / replace based on the names 👍🏻